.github/workflows/integration-build-product.yml

```diff
@@ -603,6 +603,24 @@ jobs:
         if: always()
         shell: bash
         run: |
+          set -e
+          movai-cli apt_cache stop
+
+          robots_str=$(movai-cli robots list)
+          readarray -t robots <<< "$robots_str"
+
+          if [ "${#robots[@]}" -gt 1 ]; then
+            echo "There is more than one robot. There was a bad cleanup somewhere."
+            exit 1
+          fi
+
+          movai-cli restart "${robots[0]}"
+          sleep 3
+
+          wget https://movai-scripts.s3.amazonaws.com/wait_robot_start.bash
```

Comment:
there is a PR with this feature see https://github.com/MOV-AI/movai-service/pull/376 which would be a more maintainable way to achieve synchronous  calls

Reply:
it seems its still a work in progress. lets switch it once its done.